tests/src/test/scala/whisk/core/containerpool/test/ContainerPoolTests.scala

```diff
@@ -381,6 +446,6 @@ class ContainerPoolObjectTests extends FlatSpec with Matchers with MockFactory {
             'largeYoung -> freeWorker(warmedData(namespace = largeNamespace, lastUsed = Instant.ofEpochMilli(3))),
             'largeOld -> freeWorker(warmedData(namespace = largeNamespace, lastUsed = Instant.ofEpochMilli(2))))
 
-        ContainerPool.remove(myData.invocationNamespace, pool) shouldBe Some('largeOld)
+        ContainerPool.remove(myData.action, EntityName("anyNamespace"), pool) shouldBe Some('largeOld)
```

Comment:
Same comment as above. Could use `createAction()` as there is no direct relationship to the data.